src/main/java/backend/resource/TurboLabel.java

```diff
@@ -34,12 +40,14 @@ public TurboLabel(String repoId, String name) {
         this.actualName = name;
         this.colour = "ffffff";
         this.repoId = repoId;
+        this.isInExclusiveGroup = checkIfInExclusiveGroup();
```

Comment:
Instead of a boolean, may be have an `enum` called `Grouping` with the values {`EXCLUSIVE`, `NON-EXCLUSIVE`, `NONE`}?